examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1096 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59a80aea",
+   "metadata": {},
+   "source": [
+    "## Significant advantage on large problems"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86fd991e",
+   "metadata": {},
+   "source": [
+    "We would also like to confirm how well the optimizer works for a larger system. For that, we do the following stress test with a larger graph of 22 nodes and 44 edges. QN-SPSA needs 30 sec per step, and 3 hours overall to finish the optimization (single trace). On the other hand, it takes a GD optimizer more than 13 min to finish a single optimization step for a problem with this size. We can see that the analytical gradient estimation quickly become impractical when the system size gets larger. This is the regime where QN-SPSA demonstrates its full strength."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "91d25222",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The execution time of the Job is about 3 hours, and charges about $0.4.\n",
+    "# Please confirm the budget before uncommenting and running it.\n",
+    "\"\"\"\n",
+    "n_qubits = 22\n",
+    "depth = 2\n",
+    "edges = 44\n",
+    "shots = 8192\n",
+    "load_init_config = False\n",
+    "max_iter = 300\n",
+    "learn_rate = 1e-2\n",
+    "seed = 197\n",
+    "\n",
+    "\n",
+    "hyperparameters = {\n",
+    "    \"n_qubits\": n_qubits,\n",
+    "    \"edges\": edges,\n",
+    "    \"depth\": depth,\n",
+    "    \"shots\": shots,\n",
+    "    \"seed\": seed,\n",
+    "    \"load_init_config\": load_init_config,\n",
+    "    \"max_iter\": max_iter,\n",
+    "    \"learn_rate\": learn_rate,\n",
+    "}\n",
+    "\n",
+    "instance_config = InstanceConfig(instanceType='ml.m5.2xlarge', volumeSizeInGb=30, instanceCount=1)\n",
+    "\n",
+    "\n",
+    "\n",
+    "job_name = f\"qaoa-stress-test-qubits{n_qubits}-\" + str(int(time.time()))\n",
+    "\n",
+    "\n",
+    "jobs = AwsQuantumJob.create(\n",
+    "    device=\"local:pennylane/lightning.qubit\",\n",
+    "    #device=\"arn:aws:braket:::device/quantum-simulator/amazon/sv1\",\n",
+    "    source_module=\"source_scripts\",\n",
+    "    entry_point=\"source_scripts.qaoa_stress_test\",\n",
+    "    job_name=job_name,\n",
+    "    hyperparameters=hyperparameters,\n",
+    "    instance_config=instance_config,\n",
+    "    image_uri=image_uri,\n",
+    "    wait_until_complete=False,\n",
+    ")\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5edb9ed7",
+   "metadata": {},
+   "source": [
+    "The (single-trace) optimization result is shown in the following paragraph:"
```

Comment:
Can we change the code to only run a single trace? Or is that what it is doing? the hyperparameter dict has dropped the spsa-repeat keyword as far as I can tell

Reply:
```suggestion
    "Let's take a look at the results:"
```

Reply:
Yes, it is a single trace result.